gpu/octree/src/cuda/octree_builder.cu

```diff
@@ -343,23 +357,24 @@ void pcl::device::OctreeImpl::build()
 
         {
             PointType atmax, atmin;
-            atmax.x = atmax.y = atmax.z = FLT_MAX;
-            atmin.x = atmin.y = atmin.z = -FLT_MAX;
-            atmax.w = atmin.w = 0;
+            atmax.p.x = atmax.p.y = atmax.p.z = FLT_MAX;
+            atmin.p.x = atmin.p.y = atmin.p.z = -FLT_MAX;
+            atmax.p.w = atmin.p.w = 0;
 
             //ScopeTimer timer("reduce"); 
             PointType minp = thrust::reduce(beg, end, atmax, SelectMinPoint<PointType>());
             PointType maxp = thrust::reduce(beg, end, atmin, SelectMaxPoint<PointType>());
 
-            octreeGlobal.minp = make_float3(minp.x, minp.y, minp.z);
-            octreeGlobal.maxp = make_float3(maxp.x, maxp.y, maxp.z);
+            octreeGlobal.minp = make_float3(minp.p.x, minp.p.y, minp.p.z);
+            octreeGlobal.maxp = make_float3(maxp.p.x, maxp.p.y, maxp.p.z);
         }
     		
         device_ptr<int> codes_beg(codes.ptr());
         device_ptr<int> codes_end = codes_beg + codes.size();
         {
             //ScopeTimer timer("morton"); 
-	        thrust::transform(beg, end, codes_beg, CalcMorton(octreeGlobal.minp, octreeGlobal.maxp));
+            callMorton<PointType> call(octreeGlobal.minp, octreeGlobal.maxp);
+	        thrust::transform(beg, end, codes_beg, call);
```

Comment:
indentation seems wrong

Reply:
yes, sorry - thanks for pointing it out!

Reply:
yes, I agree! Thanks